tests/blackbox/data/validation/kyma_serverless_function_not_ready.yml

```diff
@@ -0,0 +1,49 @@
+- description: This mock response explains the pod is not ready because the function is configured with 0 replicas.
+  scenario_id: kyma-serverless-function-not-ready
+  mock_response_content: |-
+    "The pod is not ready because the function is configured with 0 replicas."
```

Comment:
```suggestion
    "The function is not ready because it is configured with 0 replicas."
```

Reply:
As mention above, the function itself shows no errors. 

<img width="1134" alt="Screenshot 2024-09-11 at 21 44 49" src="https://github.com/user-attachments/assets/9da267e3-852b-4b3b-b0cf-68b77bb20b26">

However, I can add another acceptable validation, as the deployment also isn't ready (`0/0 replicas`)

<img width="1128" alt="Screenshot 2024-09-11 at 21 49 23" src="https://github.com/user-attachments/assets/8078fe43-62e3-4133-87cc-35cb1f872143">